src/core_plugins/kibana/public/context/components/size_picker/size_picker.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+import uiModules from 'ui/modules';
+import contextSizePickerTemplate from './size_picker.html';
+import './size_picker.less';
+
+const module = uiModules.get('apps/context', [
+  'kibana',
+]);
+
+module.directive('contextSizePicker', function ContextSizePicker() {
+  return {
+    bindToController: true,
+    controller: ContextSizePickerController,
+    controllerAs: 'vm',
```

Comment:
Yes, I prefer that too... then we should make sure to not use `vm` all over the new angular style guide.